performance-metrics/src/performance_tests.rs

```diff
@@ -518,6 +518,7 @@ pub fn performance_restore_latency(control: &PerformanceTestControl) -> f64 {
         ));
 
         let _ = child.kill();
+        child.wait().unwrap();
```

Comment:
Should we stick to `.unwrap()`, the same below :)

Reply:
It's kind of all over the place - I tried to adopt to what it was similar to in the same context. There is definitely some scope for cleaning this up in the future.